lib/Lift/LiftPipe.cpp

```diff
@@ -44,6 +44,9 @@ void Lift::run(Context &Ctx,
   PM.add(new LoadBinaryWrapperPass(Buffer->getBuffer()));
   PM.add(new LiftPass);
   PM.run(TargetsList.getModule());
+
+  for (auto &Target : TargetsList.enumerate())
+    Ctx.commit(Target, TargetsList);
```

Comment:
the issue with this one is that i don't have a nice way to keep it working if the root target should ever change, say because it changed the name or the kind or something similar. We probably need to take a global decision about this since it will be relevant in other pipes too.

Reply:
I'm not sure I understand your answer.